src/test/scala/nl.knaw.dans.easy.deposit/servlets/RichArchiveInputStreamSpec.scala

```diff
@@ -41,23 +41,36 @@ class RichArchiveInputStreamSpec extends TestSupportFixture {
 
   "unpackPlainEntriesTo" should "report invalid content" in {
     mockRichFileItemGetZipInputStream(new ByteArrayInputStream("Lorem ipsum est".getBytes(StandardCharsets.UTF_8))).apply(
-      unzip(_) shouldBe Failure(MalformedArchiveException("No entries found."))
+      unzip(_) shouldBe Failure(MalformedArchiveException("No entries found in uploaded.filename."))
     )
     stagingDir.entries shouldBe empty
   }
 
   it should "complain about an invalid zip" in {
     val archiveFile = "src/test/resources/manual-test/invalid.zip"
     mockRichFileItemGetZipInputStream(new FileInputStream(archiveFile)).apply(
-      unzip(_) shouldBe Failure(MalformedArchiveException("No entries found."))
+      unzip(_) shouldBe Failure(MalformedArchiveException("No entries found in uploaded.filename."))
     )
     stagingDir.entries shouldBe empty
   }
 
+  it should "complain about an invalid item" in {
+    // logs: actual and claimed size don't match ... see ...
+    // from https://github.com/apache/commons-compress/blob/f5d0bb1e287038de05415ca65145d82166a7bf0f/src/test/resources/bla-stored-dd-contradicts-actualsize.zip
+    val archiveFile = "src/test/resources/manual-test/bla-stored-dd-contradicts-actualsize.zip"
+    mockRichFileItemGetZipInputStream(new FileInputStream(archiveFile)).apply(
+      unzip(_) should matchPattern {
```

Comment:
can't you first close the brace (`)`) and then do the `should matchPattern`?
```suggestion
      unzip(_)) should matchPattern {
```